InsertRecordTimestampHeaders.md

```diff
@@ -0,0 +1,101 @@
+# Insert Wallclock
+
+## Description
+
+A Kafka Connect Single Message Transform (SMT) that inserts date, year, month,day, hour, minute and second headers using
+the record timestamp. If the record timestamp is null, the SMT uses the current system time.
+
+The headers inserted are of type STRING. By using this SMT, you can partition the data by `yyyy-MM-dd/HH`
+or `yyyy/MM/dd/HH`, for example, and only use one SMT.
+
+The list of headers inserted are:
+
+* date
+* year
+* month
+* day
+* hour
+* minute
+* second
+
+All headers can be prefixed with a custom prefix. For example, if the prefix is `wallclock_`, then the headers will be:
+
+* wallclock_date
+* wallclock_year
+* wallclock_month
+* wallclock_day
+* wallclock_hour
+* wallclock_minute
+* wallclock_second
+
+When used with the Lenses connectors for S3, GCS or Azure data lake, the headers can be used to partition the data.
+Considering the headers have been prefixed by `_`, here are a few KCQL examples:
+
+```
+connect.s3.kcql=INSERT INTO $bucket:prefix SELECT * FROM kafka_topic PARTITIONBY _date, _hour
```

Comment:
Wouldn't this be:

nnect.s3.kcql=INSERT INTO $bucket:prefix SELECT * FROM kafka_topic PARTITIONBY `_headers._date`, `_header._hour`

based on our docs:
https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sinks/s3sinkconnector/#partitions-from-message-headers